packages/block-library/src/comments-pagination-next/block.json

```
@@ -38,6 +38,23 @@
 		},
 		"interactivity": {
 			"clientNavigation": true
+		},
+		"spacing": {
+			"margin": true,
+			"padding": true
+		},
```

```suggestion
		"spacing": {
			"margin": true,
			"padding": true,
			"__experimentalDefaultControls": {
				"margin": false,
				"padding": false
			}
		},
```

Spacing controls show by default but their application here is a bit of a niche use case. To make them hidden by default, they need to be explicitly opted out of.